### PR TITLE
Feature/dx 402 fix seamless view

### PR DESCRIPTION
**Pull request by trieutran91**

### CODE REVIEW

File: _includes/payment-resource.md [hunk 1]
```diff
@@ -117,8 +117,8 @@ the `rel` and the request that will be sent in the HTTP body of the request
...
-| `update-payment-abort`        | [Aborts][abort] the payment order before any financial transactions are performed.                                                                                                                                                                               |
-| `redirect-authorization`            | Contains the URI that is used to redirect the consumer to the PayEx Payment Pages containing the card authorization UI.                                                                                                                                                                 |
-| `view-authorization`                | Contains the JavaScript href that is used to embed  the card authorization UI directly on the webshop/merchant site                                                                                                                                                                |
+| Operation                | Description                                                                                                             |
```
Comment (NullableInt) [inline _includes/payment-resource.md]: Auto-format this table.

File: payments/credit-card/redirect.md [hunk 2]
```diff
@@ -151,55 +154,55 @@ Content-Type: application/json
...
+|  ✔︎︎︎︎︎  | operation                                 | string      | Purchase                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
```
Comment (asbjornu) [inline payments/credit-card/redirect.md]: All code should be fenced. We also need a description of each property – `Purchase` isn't enough.

```suggestion
|  ✔︎︎︎︎︎  | `operation`                               | `string`    | `Purchase`                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        |
```

File: payments/credit-card/redirect.md [hunk 3]
```diff
@@ -151,55 +154,55 @@ Content-Type: application/json
...
+|  ✔︎︎︎︎︎  | prices.vatAmount                          | integer     | If the amount given includes VAT, this may be displayed for the user in the payment page (redirect only). Set to 0 (zero) if this is not relevant.                                                                                                                                                                                                                                                                                                                                                                                                                                                                |
```
Comment (asbjornu) [inline payments/credit-card/redirect.md]: `prices.vatAmount` should be replaced with the indent arrow used elsewhere and `vatAmount` without the `prices.` prefix (which should be described separately).

File: payments/credit-card/seamless-view.md [hunk 4]
```diff
@@ -65,62 +66,272 @@ options:
...
+                "vatAmount": 0
+            }
+        ],
+        "description": "Test Purchase",
+        "generatePaymentToken": "false",
+        "generateRecurrenceToken": "false",
+        "userAgent": "Mozilla/5.0...",
+        "language": "sv-SE",
+        "urls": {
+            "hostUrls": ["http://test-dummy.net"],
```
Comment (asbjornu) [inline payments/credit-card/seamless-view.md]: Replace all `test-dummy.net` URLs with `example.com`.

File: payments/credit-card/seamless-view.md [hunk 5]
```diff
@@ -65,62 +66,272 @@ options:
...
+        "intent": "Authorization",
+        "currency": "SEK",
+        "prices": [{
+                "type": "CreditCard",
+                "amount": 1500,
+                "vatAmount": 0
+            }
+        ],
+        "description": "Test Purchase",
+        "generatePaymentToken": "false",
```
Comment (asbjornu) [inline payments/credit-card/seamless-view.md]: Use literal booleans in code examples.

```suggestion
        "generatePaymentToken": false,
```

File: payments/credit-card/seamless-view.md [hunk 6]
```diff
@@ -65,62 +66,272 @@ options:
...
+                      “SCA”) on card payments, it's recommended that you add as
+                      much data as possible to the `riskIndicator` object in
+                      the request below." %}
+
+{:.code-header}
+**Request**
+
+```http
+POST /psp/creditcard/payments HTTP/1.1
+Authorization: Bearer <MerchantToken>
```
Comment (asbjornu) [inline payments/credit-card/seamless-view.md]: Replace all occurrences of `MerchantToken` with `AccessToken`.
```suggestion
Authorization: Bearer <AccessToken>
```

File: payments/credit-card/seamless-view.md [hunk 7]
```diff
@@ -141,7 +352,7 @@ sequenceDiagram
     Payer ->>Payer: Initiate iFrame
     Payer->>+Merchant: access merchant page
     deactivate Payer
-    Merchant->>+SwedbankPay: GET /psp/creditcard/payments/<paymentorder.id>
+    Merchant->>+SwedbankPay: GET /psp/creditcard/payments/<payment.id>
```
Comment (asbjornu) [inline payments/credit-card/seamless-view.md]: Let's not use `<payment.id>` in a path like this, as the Payment ID is a full URL and we don't want to encourage URL building in the client.

```suggestion
    Merchant->>+SwedbankPay: GET <payment.id>
```

File: payments/credit-card/seamless-view.md [hunk 8]
```diff
@@ -65,62 +66,270 @@ options:
...
+|  ✔︎︎︎︎︎  | `currency`                            | `string`      | NOK, SEK, DKK, USD or EUR.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |
```
Comment (asbjornu) [inline payments/credit-card/seamless-view.md]: `operation`, `intent` and `currency` are a sub-properties of `payment` so they too should have an "indent arrow", no?

File: payments/credit-card/seamless-view.md [hunk 9]
```diff
@@ -65,62 +66,270 @@ options:
...
+
+{:.code-header}
+**Response**
+
+```http
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+{
+  "payment": {
```
Comment (asbjornu) [inline payments/credit-card/seamless-view.md]: Seems like this code example only has 2 spaces indentation. Change to 4 spaces in VS Code and reformat it with the `prettyJson` extension.

File: payments/credit-card/seamless-view.md (hunk 8 above)
Comment (trieutran91) [reply payments/credit-card/seamless-view.md]: They are indeed!

**Pull request merged**